homeassistant/components/nanoleaf/device_trigger.py

```diff
@@ -52,6 +61,44 @@ async def async_get_triggers(
     ]
 
 
+async def async_get_trigger_capabilities(
+    hass: HomeAssistant, config: ConfigType
+) -> dict[str, vol.Schema]:
+    """List trigger capabilities."""
+    device_registry = dr.async_get(hass)
+    device_entry = device_registry.async_get(config[CONF_DEVICE_ID])
+    if device_entry is None:
+        raise DeviceNotFound(f"Device ID {config[CONF_DEVICE_ID]} is not valid")
+    if config[CONF_TYPE] in TRIGGER_TYPES_THAT_REPORT_PANEL_ID:
+        if nanoleaf := get_nanoleaf_connection_by_device_id(
+            hass, config[CONF_DEVICE_ID]
+        ):
+            if panels := nanoleaf.panels:
+                options = [
+                    selector.SelectOptionDict(
+                        value=str(p.id), label=f"{p.id} - {p.shape.name}"
+                    )
+                    for p in panels
+                    if p.id is not None
+                ]
+                return {
+                    "extra_fields": vol.Schema(
+                        {
+                            vol.Optional(NANOLEAF_PANEL_ID): selector.SelectSelector(
+                                selector.SelectSelectorConfig(
+                                    mode=selector.SelectSelectorMode.DROPDOWN,
+                                    multiple=False,
+                                    custom_value=True,
+                                    options=options,
+                                )
+                            )
+                        }
+                    )
+                }
+
+    return {}
```

Comment:
Other integrations call a base implementation of `async_get_trigger_capabilities`; however, doing so here adds a "duration" or "timestamp" field that the existing Nanoleaf device triggers do not support.